Sample Apps/Storefront/Storefront/CartViewController.swift

```diff
@@ -25,13 +25,16 @@
 //
 
 import UIKit
+import Pay
 
 class CartViewController: ParallaxViewController {
 
     @IBOutlet fileprivate weak var tableView: UITableView!
     
     private var totalsViewController: TotalsViewController!
     
+    fileprivate let payController = PayController(merchantID: "greats-clone.myshopify.com")
```

Comment:
Yeah I was thinking a `Info.plist` entry but that will be addressed in future PRs.